hangman_l.rb

```diff
@@ -0,0 +1,174 @@
+require "colorize"
+
+class Hangman
+
+  def initialize
+    @word_choices = ["cheeseburger", "cat", "dog", "whatever", "mississippi", "octothorpe", "splat", "bang"]
+    @answer = populate_answer
+    @wrong_counter = 0
+    @wrong_answers = []
+    @blank_answer_row = "_ " * @answer.length
+    @answer_output = @blank_answer_row.split(' ')
+
+  # Comment line below to show/hide answer
+    #puts @answer
+
+    puts @blank_answer_row
+    user_prompt
+  end
+
+  #randomly selects a word from the list
+  def populate_answer
+    random = @word_choices[rand(0...@word_choices.length)]
```

Comment:
Nice implementation of choosing a different word each time! There is actually a method on array that allows you to choose a random element from that array. Can you find it in the documentation?